docs/mesos/ManualInstallCalicoMesos.md

```diff
@@ -0,0 +1,80 @@
+<!--- master only -->
+> ![warning](../images/warning.png) This document applies to the HEAD of the calico-containers source tree.
+>
+> View the calico-containers documentation for the latest release [here](https://github.com/projectcalico/calico-containers/blob/v0.18.0/README.md).
+<!--- else
+> You are viewing the calico-containers documentation for release **release**.
+<!--- end of master only -->
+
+# Manually Install Calico and Mesos on Agent
+This guide explains how to add Calico networking to a Mesos slave.
+Specifically, we will add the Calico plugin to perform networking
+for the unified containerizer.
+- For information on the Docker Containerizer, see
+[Docker Containerizer Guide](UsageGuideDockerContainerizer.md).
+
```

Comment:
Points 1 and 2, got it.

Point 3, are you asking if we could merge the two Usage guides if they are similar enough?